README.md

```diff
@@ -1,2 +1,51 @@
 # robotont_demos
 This repository is a ROS package that contains various demos showing the capabilities of the Robotont platform
+
+## Setting up the connection between the robot and your pc
+### 1. Connect your pc to Robotont's hotspot or the same network Robotont is connected to.
+### 2. Configure Robotont's side of the network:
```

Comment:
See samm tundub üleliigne. Minu arust tuleks kustutada. @Veix123 vaata ka igaks juhuks üle

Reply:
Kui /etc/hosts failis (nii robotil kui kasutaja PC-s) on aadressid kirjas, siis ei ole seda seadistust vaja jah, aga mulle tundub, et nende käskude mõte on liikuda nimepõhiselt lahenduselt ainult IP põhisele lahendusele, et vältida hosts seadistamist. Sellisel juhul peaks need seadma hoopis .bashrc-s, et ka service kaudu käivitatud bringup laseks peremehe õigesti käima.
Lühidalt on IP aadressi kaudu ühendumine konkreetsem, ei vaja nimede lahendamist ega hosts failide muutmist. IP suureks probleemiks on aga see, kui keegi aadressiga seotud liidese maha võtab (näiteks wifi katkeb või etherneti kaabel tagant ära kukub). Juhul, kui IP kaob, siis robotisisesed sõlmed enam suhelda ei saa! Ehk pihta saab funktsionaalsus, mis võib olla võrku üldse ei vajaks... Sel põhjusel ma kasutaks (vähemalt roboti poolel) ROS_MASTER_URI vaikeväärtust (localhost) ja lisaks ühekordse /etc/hosts seadistamise READMEsse

hostname vs IP kohta ootan kindlasti ka @RValner'i arvamust!

//PS! ROS2 saves the day here